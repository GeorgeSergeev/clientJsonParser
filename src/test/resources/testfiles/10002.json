{
	"clientId": 10002,
	"subscribers": [{
			"id": 2000001,
			"spent": 1131
		} ,{
			"id": 2000002,
			"spent": 1132
		} ,{
			"id": 2000003,
			"spent": 1133
		} ,{
			"id": 2000004,
			"spent": 1134
		} ,{
			"id": 2000005,
			"spent": 1135
		} ,{
			"id": 2000006,
			"spent": 1136
		} ,{
			"id": 2000007,
			"spent": 1137
		} ,{
			"id": 2000008,
			"spent": 1138
		} ,{
			"id": 2000009,
			"spent": 1139
		} ,{
			"id": 20000010,
			"spent": 11310
		} ,{
			"id": 20000011,
			"spent": 11311
		} ,{
			"id": 20000012,
			"spent": 11312
		} ,{
			"id": 20000013,
			"spent": 11313
		} ,{
			"id": 20000014,
			"spent": 11314
		} ,{
			"id": 20000015,
			"spent": 11315
		} ,{
			"id": 20000016,
			"spent": 11316
		} ,{
			"id": 20000017,
			"spent": 11317
		} ,{
			"id": 20000018,
			"spent": 11318
		} ,{
			"id": 20000019,
			"spent": 11319
		} ,{
			"id": 20000020,
			"spent": 11320
		} ,{
			"id": 20000021,
			"spent": 11321
		} ,{
			"id": 20000022,
			"spent": 11322
		} ,{
			"id": 20000023,
			"spent": 11323
		} ,{
			"id": 20000024,
			"spent": 11324
		} ,{
			"id": 20000025,
			"spent": 11325
		} ,{
			"id": 20000026,
			"spent": 11326
		} ,{
			"id": 20000027,
			"spent": 11327
		} ,{
			"id": 20000028,
			"spent": 11328
		} ,{
			"id": 20000029,
			"spent": 11329
		} ,{
			"id": 20000030,
			"spent": 11330
		} ,{
			"id": 20000031,
			"spent": 11331
		} ,{
			"id": 20000032,
			"spent": 11332
		} ,{
			"id": 20000033,
			"spent": 11333
		} ,{
			"id": 20000034,
			"spent": 11334
		} ,{
			"id": 20000035,
			"spent": 11335
		} ,{
			"id": 20000036,
			"spent": 11336
		} ,{
			"id": 20000037,
			"spent": 11337
		} ,{
			"id": 20000038,
			"spent": 11338
		} ,{
			"id": 20000039,
			"spent": 11339
		} ,{
			"id": 20000040,
			"spent": 11340
		} ,{
			"id": 20000041,
			"spent": 11341
		} ,{
			"id": 20000042,
			"spent": 11342
		} ,{
			"id": 20000043,
			"spent": 11343
		} ,{
			"id": 20000044,
			"spent": 11344
		} ,{
			"id": 20000045,
			"spent": 11345
		} ,{
			"id": 20000046,
			"spent": 11346
		} ,{
			"id": 20000047,
			"spent": 11347
		} ,{
			"id": 20000048,
			"spent": 11348
		} ,{
			"id": 20000049,
			"spent": 11349
		} ,{
			"id": 20000050,
			"spent": 11350
		} ,{
			"id": 20000051,
			"spent": 11351
		} ,{
			"id": 20000052,
			"spent": 11352
		} ,{
			"id": 20000053,
			"spent": 11353
		} ,{
			"id": 20000054,
			"spent": 11354
		} ,{
			"id": 20000055,
			"spent": 11355
		} ,{
			"id": 20000056,
			"spent": 11356
		} ,{
			"id": 20000057,
			"spent": 11357
		} ,{
			"id": 20000058,
			"spent": 11358
		} ,{
			"id": 20000059,
			"spent": 11359
		} ,{
			"id": 20000060,
			"spent": 11360
		} ,{
			"id": 20000061,
			"spent": 11361
		} ,{
			"id": 20000062,
			"spent": 11362
		} ,{
			"id": 20000063,
			"spent": 11363
		} ,{
			"id": 20000064,
			"spent": 11364
		} ,{
			"id": 20000065,
			"spent": 11365
		} ,{
			"id": 20000066,
			"spent": 11366
		} ,{
			"id": 20000067,
			"spent": 11367
		} ,{
			"id": 20000068,
			"spent": 11368
		} ,{
			"id": 20000069,
			"spent": 11369
		} ,{
			"id": 20000070,
			"spent": 11370
		} ,{
			"id": 20000071,
			"spent": 11371
		} ,{
			"id": 20000072,
			"spent": 11372
		} ,{
			"id": 20000073,
			"spent": 11373
		} ,{
			"id": 20000074,
			"spent": 11374
		} ,{
			"id": 20000075,
			"spent": 11375
		} ,{
			"id": 20000076,
			"spent": 11376
		} ,{
			"id": 20000077,
			"spent": 11377
		} ,{
			"id": 20000078,
			"spent": 11378
		} ,{
			"id": 20000079,
			"spent": 11379
		} ,{
			"id": 20000080,
			"spent": 11380
		} ,{
			"id": 20000081,
			"spent": 11381
		} ,{
			"id": 20000082,
			"spent": 11382
		} ,{
			"id": 20000083,
			"spent": 11383
		} ,{
			"id": 20000084,
			"spent": 11384
		} ,{
			"id": 20000085,
			"spent": 11385
		} ,{
			"id": 20000086,
			"spent": 11386
		} ,{
			"id": 20000087,
			"spent": 11387
		} ,{
			"id": 20000088,
			"spent": 11388
		} ,{
			"id": 20000089,
			"spent": 11389
		} ,{
			"id": 20000090,
			"spent": 11390
		} ,{
			"id": 20000091,
			"spent": 11391
		} ,{
			"id": 20000092,
			"spent": 11392
		} ,{
			"id": 20000093,
			"spent": 11393
		} ,{
			"id": 20000094,
			"spent": 11394
		} ,{
			"id": 20000095,
			"spent": 11395
		} ,{
			"id": 20000096,
			"spent": 11396
		} ,{
			"id": 20000097,
			"spent": 11397
		} ,{
			"id": 20000098,
			"spent": 11398
		} ,{
			"id": 20000099,
			"spent": 11399
		} ,{
			"id": 200000100,
			"spent": 113100
		}
	]
}