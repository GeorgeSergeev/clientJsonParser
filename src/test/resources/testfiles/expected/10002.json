{
"clientId" : 10002,
"spentTotal" : 1145220,
"isBig" : false
}